src/PrpShop/QPlasmaUtils.cpp

```diff
@@ -606,6 +607,11 @@ std::vector<short> pqGetValidKOTypes()
     return std::vector<short>(s_typeList, s_typeList + s_numTypes);
 }
 
+bool pqIsValidKOType(short objType) {
```

Comment:
```suggestion
bool pqIsValidKOType(short objType)
{
```

Reply:
P*Shop uses different settings from the rest of the projects.  I forgot to change over. :)